desc/objectives/_coils.py

```diff
@@ -945,7 +945,7 @@ class ToroidalFlux(_Objective):
         is called on the raw compute value, before any shifting, scaling, or
         normalization. Note: has no effect for this objective
     deriv_mode : {"auto", "fwd", "rev"}
-        Specify how to compute jacobian matrix, either forward mode or reverse mode AD.
+        Specify how to compute Jacobian matrix, either forward mode or reverse mode AD.
```

Comment:
thanks for fixing all of these though in the future it might be better to do this in a separate PR if you're not otherwise modifying these files to keep things more self-contained.